db_assessment/import_db_assessment.py

```diff
@@ -689,101 +679,93 @@ def insertErrors(invalidfiles,op_df,gcpProjectName,bq_dataset):
         print ('\nWARNING: Issues while pusing Errors into operrors table with error ', pushErr)
 
 
-def populateBT(tableName,df,dataframeornot,invalidfiles,btsource,rowsimported,importresults):
+def populateBT(tableName,df,dataframeornot,invalidfiles,btsource,rowsimported,importresults,args):
     # Fuction to populate the importresults list which will be used to print using Beautiful Table 
     # rowsimported of <0 is used to indicate a FAILED status 
-    # fileName='dummy'
     tmpdataFrame=pd.DataFrame()
 
-    if btsource=='fromimportDataframeToBQ':
-
-        if dataframeornot is not None:
-            if 'PKEY' in df.columns.to_list():
-                # df2.reset_index(inplace=True)
-                if 'dbsizing_summary' not in tableName: # Need to add code to support dbsizing_summary or fix the issue with it 
+    if 'opConfig/' in invalidfiles:
+        return importresults
+    
+    if btsource=='invalidfiles': # when called from runMain
+            for fileName, error in invalidfiles.items():
+                tmpdataFrame=pd.DataFrame() 
+                tmpdataFramedict = {"Target Table":getObjNameFromFiles(fileName,'__',1),"Distinct Pkey":getObjNameFromFiles(fileName,'__',2),"Import Status":"FAILED","Loaded rows":0}
+                tmpdataFrame = tmpdataFrame.append(tmpdataFramedict, ignore_index = True)
+                if len(tmpdataFrame) >0:
+                    importresults = pd.concat([importresults, tmpdataFrame], ignore_index = True, axis = 0)
+    else:
+        if args.fromdataframe:  # when called from importDataframeToBQ
+            if dataframeornot is not None:
+                if 'PKEY' in df.columns.to_list():
+                    df.reset_index(drop=True,inplace=True) 
                     pkeygroupby=df.groupby(['PKEY']).size()
-                    # pkeygroupby=df2.groupby(level=0).size()
                     pkeycount=pkeygroupby.to_dict()
                     for pkeyname,pkeyname_rowcount in pkeycount.items():
                         if 'Elapsed' not in pkeyname:
```

Comment:
Create an issue for files with Elapsed. This should be handled in the file validation phase. Include a comment with the issue number in the code and ask the future owner of the issue to let you know when it is fixed so you can come back to it and remove the IF Elapsed.